rewrite-maven/src/main/java/org/openrewrite/maven/ChangePluginGroupIdAndArtifactId.java

```diff
@@ -35,41 +34,38 @@ public class ChangePluginGroupIdAndArtifactId extends Recipe {
     @EqualsAndHashCode.Exclude
     MavenMetadataFailures metadataFailures = new MavenMetadataFailures(this);
 
-    @Option(displayName = "Old groupId",
-            description = "The old groupId to replace. The groupId is the first part of a plugin coordinate 'com.google.guava:guava:VERSION'. Supports glob expressions.",
+    @Option(displayName = "Old group ID",
+            description = "The old group ID to replace. The group ID is the first part of a plugin coordinate 'com.google.guava:guava:VERSION'. Supports glob expressions.",
             example = "org.openrewrite.recipe")
     String oldGroupId;
 
-    @Option(displayName = "Old artifactId",
-            description = "The old artifactId to replace. The artifactId is the second part of a plugin coordinate 'com.google.guava:guava:VERSION'. Supports glob expressions.",
+    @Option(displayName = "Old artifact ID",
+            description = "The old artifactId to replace. The artifact ID is the second part of a plugin coordinate 'com.google.guava:guava:VERSION'. Supports glob expressions.",
             example = "my-deprecated-maven-plugin")
     String oldArtifactId;
 
-    @Option(displayName = "New groupId",
-            description = "The new groupId to use. Defaults to the existing group id.",
+    @Option(displayName = "New group ID",
+            description = "The new group ID to use. Defaults to the existing group ID.",
             example = "corp.internal.openrewrite.recipe",
             required = false)
     @Nullable
     String newGroupId;
 
-    @Option(displayName = "New artifactId",
-            description = "The new artifactId to use. Defaults to the existing artifact id.",
+    @Option(displayName = "New artifact ID",
+            description = "The new artifact ID to use. Defaults to the existing artifact ID.",
             example = "my-new-maven-plugin",
             required = false)
     @Nullable
-    String newArtifactId;
+    String newArtifact;
```

Comment:
Out of curiosity, was this change intended? Any existing declarative recipes that use this recipe will no longer provide the correct key, not to mention the naming is now inconsistent with other recipes like `ChangeDependencyGroupIdAndArtifactId`.

Reply:
encountered this organically upon upgrade, and I agree, it does feel like an accident

Reply:
Here's a PR that reverts the change in a backwards compatible manner
- https://github.com/openrewrite/rewrite/pull/4421